pkgs/applications/version-management/smartgithg/default.nix

```diff
@@ -37,6 +37,8 @@ stdenv.mkDerivation rec {
     mkdir -pv ${pkg_path}
     # unpacking should have produced a dir named 'smartgit'
     cp -a smartgit/* ${pkg_path}
+    # prevent using packaged jre
+    mv ${pkg_path}/jre/bin/java ${pkg_path}/jre/bin/java-unused
```

Comment:
Would removing the whole jre work or just symlinking our jre inplace to that path?

Reply:
Yes, the whole `jre/` directory can be removed. I will update it.